web/client/epics/__tests__/featuregrid-test.js

```diff
@@ -1587,14 +1591,18 @@ describe('featuregrid Epics', () => {
         };
         testEpic(autoReopenFeatureGridOnFeatureInfoClose, 1, [openFeatureGrid(), featureInfoClick(), hideMapinfoMarker(), closeFeatureGrid()], epicResult );
     });
-
-    it('autoReopenFeatureGridOnFeatureInfoClose: cancel ability to reopen feature grid on drawer toggle control', done => {
+    it('autoReopenFeatureGridOnFeatureInfoClose flow restarts on new open feature grid ', done => {
```

Comment:
was not better to add a separate test for the loop?

is this failing without the fix?

Reply:
sorry, the commit was wrong, I didn't notice some changes that didn't have to be committed, please re-check. Thank you